### PR TITLE
Add a newsletter link in the 'new version dialog'

### PR DESCRIPTION
**Pull request by fabienrohrer**
Result:

Capture d’écran 2019-07-04 à 16 06 12

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

File: src/webots/gui/WbNewVersionDialog.cpp [hunk 1]
```diff
@@ -101,6 +101,16 @@ WbNewVersionDialog::WbNewVersionDialog() {
   previewBox->setLayout(previewLayout);
   vBoxLayout->addWidget(previewBox);
 
+  // newsletter
+  QGroupBox *newsletterBox = new QGroupBox(tr("Webots newsletter:"));
+  QVBoxLayout *newsletterLayout = new QVBoxLayout();
+  label = new QLabel(tr("Keep informed about the latest enhancements of Webots by subscribing to <a style='color: #5DADE2;' "
+                        "href='https://cyberbotics.com/news/subscribe.php'>the Webots newsletter</a>."));
```
Comment (omichel) [inline src/webots/gui/WbNewVersionDialog.cpp]:
```suggestion
                        "href='https://cyberbotics.com/news/subscribe.php'>Webots newsletter</a>."));
```

File: src/webots/gui/WbNewVersionDialog.cpp [hunk 2]
```diff
@@ -101,6 +101,16 @@ WbNewVersionDialog::WbNewVersionDialog() {
   previewBox->setLayout(previewLayout);
   vBoxLayout->addWidget(previewBox);
 
+  // newsletter
+  QGroupBox *newsletterBox = new QGroupBox(tr("Webots newsletter:"));
+  QVBoxLayout *newsletterLayout = new QVBoxLayout();
+  label = new QLabel(tr("Keep informed about the latest enhancements of Webots by subscribing to <a style='color: #5DADE2;' "
```
Comment (omichel) [inline src/webots/gui/WbNewVersionDialog.cpp]:
```suggestion
  label = new QLabel(tr("Stay informed about the latest developments of Webots by subscribing to the <a style='color: #5DADE2;' "
```

**Comment by fabienrohrer** [comment]
No it's welcome :-)

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**